### ISSUE TITLE
No module named 'importlib.metadata' for Python 3.7

### ISSUE THREAD

**Issue by o-aleks** [maurerle/eralchemy2#13]
Hello,
There is an issue with Python 3.7 and eralchemy2:
```
  File ".....lib/site-packages/eralchemy2/main.py", line 22, in <module>
    from importlib.metadata import version
ModuleNotFoundError: No module named 'importlib.metadata'
```

Is it possible to use some modifications like this:
```
try:
    from importlib.metadata import version
except ImportError: # for Python<3.8
    from importlib_metadata import version
``` 
Thank you.

**Comment by maurerle** [comment]
This is already fixed on the main branch (https://github.com/maurerle/eralchemy2/commit/6f35f4a69a3d9de2aebf4a06ccd8e5b1bfd7ce17), but I did not release yet, sorry.
Will do it today

**Comment by o-aleks** [comment]
Thank you, @maurerle! 
BTW is there some issue with releasing it?

Regards.

**Comment by maurerle** [comment]
Waited for the pipeline but forgot to push to pypi then..
I just did, so its available on pypi as 1.3.5.

**Comment by o-aleks** [comment]
Thank you!!
